Common/UI/Armor/UIGearInventory.cs

```diff
@@ -357,7 +360,7 @@ private static UIElement BuildDefaultInventory()
 		necklace.OnMouseOver += UpdateMouseOver;
 		necklace.OnMouseOut += UpdateMouseOut;
 
-		necklace.Predicate = (item, _) => item.accessory && item.wingSlot <= 0;
+		necklace.Predicate = (item, _) => item.ModItem is Amulet;
```

Comment:
Hypothetical; we may want a way for other mods to designate an item is an amulet (or offhand, ring...) *without* it strictly needing a ModItem to match this condition. A HashSet<int> somewhere for a lookup would be all we need in the future.